README.md

```diff
@@ -0,0 +1,85 @@
+![OpenWrt logo](/logo.svg)
+
+OpenWrt Project is a Linux operating system targeting embedded devices. Instead
+of trying to create a single, static firmware, OpenWrt provides a fully
+writable filesystem with package management. This frees you from the
+application selection and configuration provided by the vendor and allows you
+to customize the device through the use of packages to suit any application.
+For developers, OpenWrt is the framework to build an application without having
+to build a complete firmware around it; for users this means the ability for
+full customization, to use the device in ways never envisioned. 
+
+Latest release: **19.07.3**
```

Comment:
I'd remove the stable release, I don't think this is the place to put this information (particularly by introducing a change to master).

Reply:
Yea, most other projects use some fancy embedded buttons. I like the idea of having a version information but maybe the GitHub/GitLab releases overview is enough for that.